tests/Makefile

```diff
@@ -246,3 +247,17 @@ include/test/instrument_record_status.h: ../include/psa/crypto.h Makefile
 	echo "  Gen  $@"
 	sed <../include/psa/crypto.h >$@ -n 's/^psa_status_t \([A-Za-z0-9_]*\)(.*/#define \1(...) RECORD_STATUS("\1", \1(__VA_ARGS__))/p'
 endif
+
+libpsaclient libpsaserver:
+	# Clone the library and include folder for client and server builds.
+	rm -Rf ./$@
```

Comment:
(minor) consistency: why not use `-rf` like at like 176? (Yes, I'm aware we sometimes seem to use `-r` and sometimes `-R` for no very good reason)